testapi.pm

```diff
@@ -812,8 +788,7 @@ Return the given variable as array reference (split variable value by , | or ; )
 
 =cut
 
-sub get_var_array {
-    my ($var, $default) = @_;
+sub get_var_array ($var, $default) {
```

Comment:
ok, this should have been obvious. Failed in https://openqa.opensuse.org/tests/2359937

```
Too few arguments for subroutine 'testapi::get_var_array' at /usr/lib/os-autoinst/testapi.pm line 807.
	testapi::get_var_array("SCC_ADDONS") called at /usr/lib/os-autoinst/testapi.pm line 807
	testapi::check_var_array("SCC_ADDONS", "tcm") called at opensuse/lib/main_common.pm line 1288
	main_common::load_consoletests() called at opensuse/products/opensuse/main.pm line 387
	require opensuse/products/opensuse/main.pm called at /usr/lib/os-autoinst/OpenQA/Isotovideo/Utils.pm line 196
	OpenQA::Isotovideo::Utils::try {...} () called at /usr/lib/perl5/vendor_perl/5.26.1/Try/Tiny.pm line 102
	eval {...} called at /usr/lib/perl5/vendor_perl/5.26.1/Try/Tiny.pm line 93
	Try::Tiny::try(CODE(0x563d02a976a0), Try::Tiny::Catch=REF(0x563d01e52150)) called at /usr/lib/os-autoinst/OpenQA/Isotovideo/Utils.pm line 213
	OpenQA::Isotovideo::Utils::load_test_schedule() called at /usr/bin/isotovideo line 245
Compilation failed in require at /usr/lib/os-autoinst/OpenQA/Isotovideo/Utils.pm line 196.
	OpenQA::Isotovideo::Utils::try {...} () called at /usr/lib/perl5/vendor_perl/5.26.1/Try/Tiny.pm line 102
	eval {...} called at /usr/lib/perl5/vendor_perl/5.26.1/Try/Tiny.pm line 93
	Try::Tiny::try(CODE(0x563d02a976a0), Try::Tiny::Catch=REF(0x563d01e52150)) called at /usr/lib/os-autoinst/OpenQA/Isotovideo/Utils.pm line 213
	OpenQA::Isotovideo::Utils::load_test_schedule() called at /usr/bin/isotovideo line 245

 at /usr/lib/os-autoinst/OpenQA/Isotovideo/Utils.pm line 212.
	OpenQA::Isotovideo::Utils::catch {...} ("Too few arguments for subroutine 'testapi::get_var_array' at "...) called at /usr/lib/perl5/vendor_perl/5.26.1/Try/Tiny.pm line 123
	Try::Tiny::try(CODE(0x563d02a976a0), Try::Tiny::Catch=REF(0x563d01e52150)) called at /usr/lib/os-autoinst/OpenQA/Isotovideo/Utils.pm line 213
	OpenQA::Isotovideo::Utils::load_test_schedule() called at /usr/bin/isotovideo line 245
5356: EXIT 1
```